src/core/logical_layers/renderers/BivariateRenderer/setTileScheme.ts

```diff
@@ -0,0 +1,9 @@
+import type { RasterSource, VectorSource } from 'maplibre-gl';
+
+/* https://docs.mapbox.com/mapbox-gl-js/style-spec/sources/#vector-scheme */
+export function setTileScheme(rawUrl: string, mapSource: VectorSource | RasterSource) {
```

Comment:
we already have it in /src/core/logical_layers/renderers/GenericRenderer.ts#L479
should we move it (and similar stuff from GenericRenderer) to utils?